### ISSUE TITLE
App crashes with DEBUG_MODEL = false

### ISSUE THREAD

**Issue by HNeukermans** [googlecodelabs/tensorflow-style-transfer-android#13]
I always get an 'input_max_range must be larger than input_min_range' exception,
when DEBUG_MODEL = false. Setting DEBUG_MODEL to true helps to solve the problem

device : moto 5G android nougat 27

06-12 20:23:31.752 29815-29834/? E/AndroidRuntime: FATAL EXCEPTION: inference
Process: org.tensorflow.demo, PID: 29815
java.lang.IllegalArgumentException: input_max_range must be larger than input_min_range.
[[Node: transformer/contract/conv2/Relu_eightbit_quantize_transformer/contract/conv2/InstanceNorm/batchnorm/add_1 = QuantizeV2[T=DT_QUINT8, mode="MIN_FIRST", _device="/job:localhost/replica:0/task:0/cpu:0"](transformer/contract/conv2/InstanceNorm/batchnorm/add_1, transformer/contract/conv2/Relu_eightbit_min_transformer/contract/conv2/InstanceNorm/batchnorm/add_1, transformer/contract/conv2/Relu_eightbit_max_transformer/contract/conv2/InstanceNorm/batchnorm/add_1)]]
at org.tensorflow.Session.run(Native Method)
at org.tensorflow.Session.access$100(Session.java:48)
at org.tensorflow.Session$Runner.runHelper(Session.java:285)
at org.tensorflow.Session$Runner.run(Session.java:235)
at org.tensorflow.contrib.android.TensorFlowInferenceInterface.run(TensorFlowInferenceInterface.java:142)
at org.tensorflow.demo.StylizeActivity.stylizeImage(StylizeActivity.java:590)
at org.tensorflow.demo.StylizeActivity.access$1400(StylizeActivity.java:67)
at org.tensorflow.demo.StylizeActivity$3.run(StylizeActivity.java:545)
at android.os.Handler.handleCallback(Handler.java:751)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:154)
at android.os.HandlerThread.run(HandlerThread.java:61)

**Comment by humayuntanwar** [comment]
Still getting the same error when debug_mode is set to false , any solutions yet?
AS 3.1.3 Phone Huawei p9 lite
setting debug_mode = true will not let us access camera, which i want to access

**Comment by freeknortier** [comment]
Same issue here. Using a Samsung Galaxy S6 with API 24
